src/client/sandbox/native-methods.js

```diff
@@ -212,6 +212,16 @@ class NativeMethods {
             this.CSS2PropertyRemoveProperty   = win.CSS2Property.prototype.removeProperty;
         }
 
+        // Console
+        this.console = win.console;
+
+        this.consoleMeths = {
```

Comment:
Why? Is there a significant reason for this? I believe it's better to leave it as is to use
```js
sandbox.nativeMethods.consoleMeths[meth]
```
instead of
```js
const nativeMeth = console + meth[0].toUpperCase() + meth.slice(1);
sandbox.nativeMethods[nativeMeth];
```

Reply:
Ok. If you use a console method as key then let you leave it as is.